task3/vector.js

```diff
@@ -0,0 +1,81 @@
+function sortAsc(array) {
+    return array.sort((a,b) => a-b);
+}
+
+class Vector {
+    constructor(inputArray) {
+        this.array = inputArray;
+    }
+
+    checkIsSameLength(vector) {
+        if (this.array.length !== vector.array.length) {
```

Comment:
You could avoid this structure `vector.array` with **destructuring assignment**
You just need to initialize the method in the following way:
`checkIsSameLength({array}) {`
This structure **`{array}`** equals **`vector.array`**, but inside the function, you could just use **array**.
Thanks, destructuring assignment :)